src/app/components/quiz-page/quiz-page.component.scss

```diff
@@ -0,0 +1,22 @@
+.content {
```

Comment:
spacje zjedzone prawie wszedzie pomiedzy nazwa klasy a nawiasem

Reply:
ok to popraiwe i domerguje